CMakeLists.txt

```diff
@@ -107,7 +107,7 @@ if(BUILD_GDASBUNDLE)
 # Build IODA converters
   option(BUILD_IODA_CONVERTERS "Build IODA Converters" ON)
   if(BUILD_IODA_CONVERTERS)
-    ecbuild_bundle( PROJECT iodaconv GIT "https://github.com/JCSDA-internal/ioda-converters.git" BRANCH develop )
```

Comment:
note to remove this change before merging

Reply:
Can we just revert this file so that it is not included in the PR?

Reply:
I just did this to revert it:

 git checkout 3f2b79d -- CMakeLists.txt
git commit -m 'revert CMakeLists.txt'
git push

anything else I need to do in order to revert it?

Reply:
nope, that's perfect, thanks! Note the 3dvar YAML will also need this done

Reply:
Thanks!